src/mpris.cpp

```diff
@@ -39,6 +42,11 @@ Mpris::~Mpris()
 {
 }
 
+QObject *Mpris::api_factory(QQmlEngine *, QJSEngine *)
+{
+    return new Mpris();
```

Comment:
We could do that but the reason for the existence of this class and not just a namespace is purely because of QML restrictions to use  enumerations.

In other words, you will never want to create an instance of this class from C++; it only has static methods and an enumeration.